datatypes/src/collections/feature_collection.rs

```diff
@@ -805,6 +805,7 @@ pub trait FeatureCollectionInfos {
     fn byte_size(&self) -> usize;
 }
 
+#[allow(dead_code)]
```

Comment:
maybe pub use it in the mod to get rid of the warning? Or remove it if we never use, or plan to use it.

Reply:
[6849724](https://github.com/geo-engine/geoengine/pull/973/commits/684972467e851106b49dca727adc36c1d5595550)